cert/CACreationScript.sh

```diff
@@ -0,0 +1,132 @@
+
+#Operator EE
+OPER_EE=wif_oper_req
+OPER_EE_SIGN=$OPER_EE$SIG
+
+openssl req -new -newkey rsa:$EE_RSA_KEY_SIZE -nodes -reqexts $OPER_EE -out $OPER_EE.csr -keyout $OPER_EE.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=Operator End-Entity Example" -config openssl.cnf
+
+openssl ca -create_serial -cert $OPER.crt -keyfile $OPER.pkey -outdir $CSR -batch -out $OPER_EE.crt -utf8 -days $EE_VALIDITY -policy policy_anything -md sha384 -extensions $OPER_EE_SIGN -config openssl.cnf -in $OPER_EE.csr
+
+#Professional Installer Intermediate
+PI=wif_pi_ca
+PI_SIGN=$PI$SIG
+
+openssl req -new -newkey rsa:$INT_RSA_KEY_SIZE -nodes -reqexts $PI -out $PI.csr -keyout $PI.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=WInnForum RSA Installer CA-1" -config openssl.cnf
+
+openssl ca -create_serial -cert $ROOT.crt -keyfile $ROOT.pkey -outdir $CSR -batch -out $PI.crt -utf8 -days $INT_VALIDITY -policy policy_anything -md sha384 -extensions $PI_SIGN -config openssl.cnf -in $PI.csr
+
+#Professional Installer EE
+PI_EE=wif_pi_req
+PI_EE_SIGN=$PI_EE$SIG
+
+openssl req -new -newkey rsa:$EE_RSA_KEY_SIZE -nodes -reqexts $PI_EE -out $PI_EE.csr -keyout $PI_EE.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=Installer End-Entity Example" -config openssl.cnf
+
+openssl ca -create_serial -cert $PI.crt -keyfile $PI.pkey -outdir $CSR -batch -out $PI_EE.crt -utf8 -days $EE_VALIDITY -policy policy_anything -md sha384 -extensions $PI_EE_SIGN -config openssl.cnf -in $PI_EE.csr
+
+#PAL: TODO
+
+#Device Intermediate
+CBSD=wif_cbsd_ca
+CBSD_SIGN=$CBSD$SIG
+
+openssl req -new -newkey rsa:$INT_RSA_KEY_SIZE -nodes -reqexts $CBSD -out $CBSD.csr -keyout $CBSD.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=WInnForum RSA Device CA-1" -config openssl.cnf
+
+openssl ca -create_serial -cert $ROOT.crt -keyfile $ROOT.pkey -outdir $CSR -batch -out $CBSD.crt -utf8 -days $INT_VALIDITY -policy policy_anything -md sha384 -extensions $CBSD_SIGN -config openssl.cnf -in $CBSD.csr
+
+#Device EE
+CBSD_EE=wif_cbsd_req
+CBSD_EE_SIGN=$CBSD_EE$SIG
+
+openssl req -new -newkey rsa:$EE_RSA_KEY_SIZE -nodes -reqexts $CBSD_EE -out $CBSD_EE.csr -keyout $CBSD_EE.pkey -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=Device End-Entity Example" -config openssl.cnf
+
+openssl ca -create_serial -cert $CBSD.crt -keyfile $CBSD.pkey -outdir $CSR -batch -out $CBSD_EE.crt -utf8 -days $EE_VALIDITY -policy policy_anything -md sha384 -extensions $CBSD_EE_SIGN -config openssl.cnf -in $CBSD_EE.csr
+
+#TODO: manufacturer intermediate signed by $CBSD, and EE cert signed by manufacturer intermediate
```

Comment:
I have this done in my next PS script, so once this PR is merged, I'll copy the functionality over to the Bash script here as well.